projects/opal-frontend-common/directives/capitalisation/capitalisation.directive.ts

```diff
@@ -19,9 +19,12 @@ export class CapitalisationDirective {
     const target = event.target as HTMLInputElement;
     const value = target.value;
 
```

Comment:
- Immediately exit if the input is empty
- No need to uppercase or fire an event for a blank input

```suggestion
if (!value) {
      return; 
    }
```